actors/builtin/market/market_actor.go

```diff
@@ -92,16 +93,16 @@ func (a Actor) WithdrawBalance(rt Runtime, params *WithdrawBalanceParams) *adt.E
 }
 
 // Deposits the received value into the balance held in escrow.
-func (a Actor) AddBalance(rt Runtime, providerOrClientAdrress *addr.Address) *adt.EmptyValue {
-	escrowAddress(rt, *providerOrClientAdrress)
+func (a Actor) AddBalance(rt Runtime, providerOrClientAddress *addr.Address) *adt.EmptyValue {
+	btabAddr := escrowAddress(rt, *providerOrClientAddress)
```

Comment:
This changes behaviour in a way you probably don't want (see https://filecoinproject.slack.com/archives/CHMNDCK9P/p1581987982474800)

This logic is obscure. It may be better to duplicate it between the power and miner actors than attempt the partial code-sharing.

Reply:
Edit: it looks like I already split the power actor's use of similar concepts, so these are specific to the market.

Reply:
I have updated this to use the code on master